barretenberg/cpp/src/barretenberg/stdlib/eccvm_verifier/eccvm_recursive_verifier.cpp

```diff
@@ -142,10 +145,12 @@ template <typename Flavor> void ECCVMRecursiveVerifier_<Flavor>::verify_proof(co
     const OpeningClaim batch_opening_claim =
         Shplonk::reduce_verification(key->pcs_verification_key->get_g1_identity(), opening_claims, transcript);
 
-    // TODO(https://github.com/AztecProtocol/barretenberg/issues/1142): Handle this return value correctly.
-    const typename PCS::VerifierAccumulator batched_opening_accumulator =
-        PCS::reduce_verify(batch_opening_claim, ipa_transcript);
+    // // TODO(https://github.com/AztecProtocol/barretenberg/issues/1142): Handle this return value correctly.
+    // const typename PCS::VerifierAccumulator batched_opening_accumulator =
```

Comment:
TODO resolved?

Reply:
yep, we no longer call the IPA recursive verifier in the eccvm recursive verifier